source/chapters/user_interface.rst

```diff
@@ -605,9 +605,9 @@ Headphone and Master Mix Controls
   .. note:: Don't forget to activate the :guilabel:`PFL` button on the deck you
             want to listen to in your headphones.
 
-**Head volume Knob**
-  This button adjusts your headphone volume. You can adjust the volume of a
-  single deck's signal you are listening to in the headphones with the
+**Head Gain Knob**
+  This button adjusts your headphone gain. You can adjust the gain of a
+  single deck's signal you are listening to in the headphones with the deck's
```

Comment:
We should add a hint, that this is not a hardware volume, which should be preferred if available.